style.css

```diff
@@ -98,3 +103,241 @@ a {
 p a {
   color: #7f8cff;
 }
+
+/* signig worection */
+.workcards {
+  display: grid;
+  row-gap: 2em;
+  padding: 100px 25px;
+}
```

Comment:
- [ ] Please make sure to use the correct background color for the project card's main container. The background color given in the Figma template is `#F7F7F9` Kindly fix this to fulfil the following requirement 👇
- [ ] Also, kindly make sure to increase the `gap` between the project cards as given in the design guidelines

  > _You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)._
  
Your design | Given design
---| ---|
![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/c14d53d7-9a0d-4e29-9530-0f94d8be0952) | ![image](https://github.com/ali-techworld/Portfolio-Setup/assets/106119105/d29e1a4f-2829-49e2-a3b7-fc83459d07da)